deflect/Stream.h

```diff
@@ -86,18 +86,22 @@ class Stream
      * isConnected().
      *
      * Different Streams can contribute to a single window by using the same
-     * name as identifier. All the Streams which contribute to the same window
-     * should be created before any of them starts sending images.
+     * identifier. All the Streams which contribute to the same window should be
+     * created before any of them starts sending images.
      *
-     * @param name An identifier for the stream which cannot be empty.
-     * @param address Address of the target Server instance, can be a
-     *                hostname like "localhost" or an IP in string format like
-     *                "192.168.1.83".
+     * @param id The identifier for the stream. If set, the environment variable
+     *           DEFLECT_ID takes precedence over the given argument.
+     *           If both values are empty, the constructor will throw.
+     * @param host The address of the target Server instance. It can be a
+     *             hostname like "localhost" or an IP in string format like
+     *             "192.168.1.83". If set, the environment variable DEFLECT_HOST
+     *             takes precedence over the given value.
```

Comment:
That's a fair point. @tribal-tec argued the other way around, that ENV variables should be able to override runtime behaviour without recompiling. Maybe command line arguments should still be able to take precedence?